fault-tolerance/src/main/java/io/helidon/faulttolerance/Retry.java

```diff
@@ -640,6 +641,387 @@ public Builder config(Config config) {
+             * </tr>
+             * <tr>
+             *     <td>calls</td>
+             *     <td>10</td>
+             *     <td>Number of calls</td>
+             * </tr>
+             * <tr>
+             *     <td>initial-delay</td>
+             *     <td>2</td>
+             *     <td>Initial delay</td>
+             * </tr>
+             * <tr>
+             *     <td>max-delay</td>
+             *     <td>3 minutes</td>
+             *     <td>Maximum delay</td>
+             * </tr>
+             * <tr>
+             *     <td>jitter</td>
+             *     <td>50 milliseconds</td>
+             *     <td>A number between {@code [-jitter,+jitter]} applied to delay</td>
+             * </tr>
+             * </table>
+             *
+             * @param config the config node to use
+             * @return updated builder instance
+             */
+            public Builder config(Config config) {
+                config.get("calls").asInt().ifPresent(this::calls);
+                config.get("initial-delay").as(Duration.class).ifPresent(this::initialDelay);
+                config.get("max-delay").as(Duration.class).ifPresent(this::maxDelay);
+                config.get("jitter").asLong().ifPresent(this::jitter);
+                return this;
+            }
+        }
+    }
+
+
+    /**
+     * A retry policy that increases the delay time following an exponential sequence.
+     * Allowed elements that are also annotated with {@code @Retry}.
+     * Expected sequence: initial delay, 2 * initial delay + jitter, 4 * initial delay + jitter,
+     * 8 * initial delay + jitter, etc. {@code maxDelayInMillis} is used to prevent endless waiting.
+     */
+    class ExponentialRetryPolicy implements RetryPolicy {
+
+        private final int calls;
+        private final Duration initialDelay;
+        private final Duration maxDelay;
+        private final int factor;
+        private final Supplier<Long> jitter;
+
+        private long accumulateDelay;
+
+
+        private ExponentialRetryPolicy(Builder builder) {
+            calls = builder.calls;
+            initialDelay = builder.initialDelay;
+            maxDelay = builder.maxDelay;
+            factor = builder.factor;
+
+            long jitter = builder.jitter;
+            int jitterMillis = (jitter > Integer.MAX_VALUE) ? Integer.MAX_VALUE : (int) jitter;
+            if (jitterMillis == 0L) {
+                this.jitter = () -> 0L;
+            } else {
+                Random random = new Random();
+                // need a number [-jitterMillis,+jitterMillis]
+                this.jitter = () -> Long.valueOf(random.nextInt((jitterMillis * 2) - jitterMillis));
+            }
+        }
+
+        /**
+         * A new fluent API builder to configure instances of {@link io.helidon.faulttolerance.Retry}.
+         *
+         * @return a new builder
+         */
+        public static Builder builder() {
+            return new Builder();
+        }
+
```

Comment:
Minor comment, there's inconsistent separation of methods/classes with different number of new lines.

Reply:
Fixed